stencil-library/storybook-pages/angular.stories.mdx

```diff
@@ -0,0 +1,189 @@
+import { Meta, Source } from '@storybook/addon-docs';
+import { SummaryElement } from './utils.tsx';
+import CSSVars from './css-variables.tsx';
+import dedent from 'ts-dedent';
+
+<Meta title="Angular" />
+
+# Justifi Angular web components
+
+# Index
+
+- [Usage](#usage)
+- [Integration Steps](#integration-steps)
+- [Props and Event Handling](#props-and-event-handling)
+- [Calling methods](#calling-methods)
+- [Styling](#styling)
+  - [Components styled with variables](#components-styled-with-variables)
+  - [Styling exported parts](#styling-exported-parts)
+
+# Usage
+
+To integrate Justifi web components into your Angular application, use the `@justifi/webcomponents` package directly. This approach allows you to utilize the standard web components in an Angular environment.
+
+## Integration Steps
+
+1. Include Web Components in index.html:
+
+<Source
+  language="bash"
+  dark
+  code={dedent(`
+    <head>
+      <link rel="stylesheet"
+        href="https://cdn.jsdelivr.net/npm/@justifi/webcomponents@4.6.0/dist/webcomponents/webcomponents.css" />
+      <script type='module'
+        src='https://cdn.jsdelivr.net/npm/@justifi/webcomponents@4.6.0/dist/webcomponents/webcomponents.esm.js'></script>
+    </head>
+  `)}
+/>
+
+2. Update Angular Module Configuration:
+
+<Source
+  language="typescript"
+  dark
+  code={dedent(`
+    import { NgModule, CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+    import { BrowserModule } from '@angular/platform-browser';
+    import { AppComponent } from './app.component';
+
+    @NgModule({
+      declarations: [
+        AppComponent
+      ],
+      imports: [
+        BrowserModule
+      ],
+      providers: [],
+      bootstrap: [AppComponent],
+      schemas: [
+        CUSTOM_ELEMENTS_SCHEMA
+      ]
+    })
+    export class AppModule { }
+
+`)}
+/>
+
+## Props and Event Handling
+
+- **Props**: Use Angular template syntax for passing props to components. For example, `validationMode` accepts `onBlur`, `onChange` or `onSubmit`. It's also possible to pass variables:
```

Comment:
Instead of `It's also possible to pass variables`, I think it might be clearer to say `You may also use expressions for prop values as with any other Angular component` or something like that.